shared/profile/prove-enter-username-container.js

```diff
@@ -35,7 +35,7 @@ export default connector.connect(
     }
 
     return {
-      canContinue: profile.usernameValid,
+      canContinue: true,
```

Comment:
i think this is actually incorrect. we do validate it , its just this logic is somehow broken. i'll fix in my recompose pr